bokehjs/src/lib/models/glyphs/ngon.ts

```diff
@@ -0,0 +1,204 @@
+import {RadialGlyph, RadialGlyphView} from "./radial_glyph"
+import type {PointGeometry, PolyGeometry, RectGeometry, SpanGeometry} from "core/geometry"
+import {minmax2} from "core/util/arrayable"
+import {check_2_segments_intersect, point_in_poly} from "core/hittest"
+import * as p from "core/properties"
+import type {Arrayable} from "core/types"
+import type {Context2d} from "core/util/canvas"
+import {Selection} from "../selections/selection"
+
+export interface NgonView extends Ngon.Data {}
+
+function ngon(x: number, y: number, r: number, n: number, angle: number) {
```

Comment:
This needs a return type because currently this is inferred as an array of f32 arrays, perhaps like this:
```suggestion
function ngon(x: number, y: number, r: number, n: number, angle: number): [Arrayable<number>, Arrayable<number>] {
```
I generally advice to always add return types, so that the API, public or not, doesn't depend on inferred types, because those may not be clear or may change when implementation changes or may change when the compiler changes.

Reply:
[36668ab](https://github.com/bokeh/bokeh/pull/14027/commits/36668abbf65c1a58216256243cbaaabc00280b6f)